### ISSUE TITLE
Add ML inference point to CLI

### ISSUE THREAD

**Issue by hmacdope** [choderalab/asapdiscovery#920]
Currently you cannot run ML inference from the command line, we should make this something that is easy to do.

**Comment by kaminow** [comment]
yeah this is definitely needed. I feel like this is something I've started a handful of times and never actually fully finished
